blog/views.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.core.paginator import Paginator
+
+from django.db.models import Count
+from django.http import HttpResponse, HttpResponseRedirect
+from django.shortcuts import render, get_object_or_404, redirect
+
+from django.urls import reverse_lazy
+
+from django.views import generic
+
+from blog.models import Post, Commentary
+
+
+def index(request) -> HttpResponse:
+    posts = (Post.objects.all()
+             .annotate(num_comments=Count("comments"))
+             .order_by("-created_time"))

```

Comment:
1. fix formatting(brackets shouldn't have some code on the same lvl)
2. .all() is redundant